pyiode/objects/table.pyx

```diff
@@ -1296,6 +1323,13 @@ cdef class Table:
         >>> # insert new line with cells
         >>> # "    -> STRING cell
         >>> # no " -> LEC cell
+        # GDM> as much as I understand this explanation, I think it is not clear
+        #      enough for inexperienced Python programmers,
+        #      as I think somebody is bound to write
+        #      >>> table.insert(index, ["RIDG:", 'RIDG'])
```

Comment:
@jmpplan 

From the French documentation of IODE: 
> Définition des cellules
> Pour distinguer une cellule texte d'une cellule LEC, la cellule de texte devra commencer par les doubles guillemets (qui ne seront pas imprimés). Une cellule ne débutant pas par un double guillemet sera considérée comme une forme LEC et syntaxiquement vérifiée.

In other words:
> Cell definition
> To distinguish a text cell from a LEC cell, the text cell must begin with double quotation marks (which will not be printed). A cell that doesn't start with a double quotation mark will be considered a syntax-checked LEC form.

I know it's weird but long-time IODE users are used to this implicit rule.